proposals/POEM-2-function-pulling-container-scheduler.md

```diff
@@ -0,0 +1,132 @@
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+# Title
+Function Pulling Container Scheduler(FPCScheduler)
```

Comment:
```suggestion
Function Pulling Container Scheduler (FPCScheduler)
```

Reply:
Small note: I'm not sure about this name - the existing scheduler is also a pull scheduler in that each invoker pulls from the kafka queue and schedules containers.

Reply:
Actually, what we tried to mean is a scheduler that schedules containers pulling functions.
So now each container pulls functions and scheduler just schedule such containers.
The existing scheduler schedules function requests. Currently, containers and function requests are identical in terms of scheduling. In my recognition, it just schedules function requests but the underlying implementation is just containers. 

But anyway it would be great if there is a better name for this intention.

Reply:
Something like `Function Definition Container Scheduler` for what the existing scheduler is now becoming. Having pulling in the name can be confusing with that it's a part of the new method.  And then something like `Activation Polling` for the new queue pulling from within the containers.